src/terminal/terminal/ansi_escape_codes.h

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+#ifndef _GUAC_TERMINAL_ANSI_ESCAPE_CODES_H
+#define __GUAC_TERMINAL_ANSI_ESCAPE_CODES_H
+
+#include "config.h"
+
+/**
+ * The maximum number of parameters the CSI can have is 16.
```

Comment:
You don't need to include the value this macro expands to within the documentation for the macro itself. Part of the point of documentation and macros like this are to avoid the need to know such implementation-specific details.

Something like "the maximum number of parameters the CSI can have" is probably almost perfect, though I recommend finding a better way to say "the CSI" that is in line with existing documentation and standards covering console codes. I've seen this referred to as "the CSI sequence" before, but never "the CSI".